gapis/perfetto/cc/processor.cpp

```diff
@@ -27,6 +27,10 @@ namespace ptp = perfetto::trace_processor;
 
 processor new_processor() {
   ptp::Config config;
+  // TODO(b/154156099): In Android S, AGI should use the time when trace starts
```

Comment:
Maybe this should be tracked in a new bug, since this PR will arguably close b/154156099? (I'm not sure about bug tracking best practices, it's a more a question than a request)

Reply:
I think we should always link to the old bug so that anyone that picks this up can have the whole picture without having to dig into a lot of logs. And then the one that picks this up can figure out whether a new bug should be filed or just work on the old bug.

Reply:
:ok_hand: